homeassistant/components/xiaomi_miio/light.py

```diff
@@ -938,3 +964,109 @@ async def async_update(self):
     async def async_set_delayed_turn_off(self, time_period: timedelta):
         """Set delayed turn off. Unsupported."""
         return
+
+
+class XiaomiGatewayLight(LightEntity):
+    """Representation of a gateway device's light."""
+
+    def __init__(self, gateway_device, gateway_name, gateway_device_id):
+        """Initialize the XiaomiGatewayLight."""
+        self._gateway = gateway_device
+        self._name = f"{gateway_name} Light"
+        self._gateway_device_id = gateway_device_id
+        self._unique_id = f"{gateway_device_id}-light"
```

Comment:
We don't need to add the platform to the unique id. The entity registry is aware of both the integration domain and the platform domain. Just make sure that different entity types for the same integration and platform are resolved with different unique id.

Reply:
done